### ISSUE TITLE
SRD0039 triggered for CTE

### ISSUE THREAD

**Issue by niphlod** [ErikEJ/SqlServer.Rules#666]
It seems that SRD0039 is triggered for CTEs

e.g.

```
WITH set1 AS (
    SELECT a1 = 'a'
)

SELECT tt.name
FROM sys.tables tt
INNER JOIN set1
ON set1.a1 = tt.name;
```

**Comment by ErikEJ** [comment]
@niphlod Thanks for reporting - fixed in latest CLI build .59
